lib/runner.js

```diff
@@ -54,7 +54,11 @@ class Runner extends Emittery {
 		let hasStarted = false;
 		let scheduledStart = false;
 		const meta = Object.freeze({
-			file: options.file
+			file: this.file,
```

Comment:
Doesn't really matter, I guess, but should I revert this to use `options` again? I switched to `this` since I think that's what's being used elsewhere, but now it looks kinda inconsistent when `snapshotDirectory()` is using `options`. Opinion?

Reply:
Yea I'll change it back.